features/admin.organizations.v1/pages/organization-roles-edit.tsx

```diff
@@ -47,7 +50,7 @@ const OrganizationRolesEdit: FunctionComponent = (): ReactElement => {
         setIsRoleDetailsRequestLoading(true);
 
         getOrganizationRoleById(currentOrganization.id, roleId)
-            .then(response => {
+            .then((response: any) => {
```

Comment:
`any` is used here because the response returned from `getOrganizationRoleById` returns `Promise<any>`.[1]

[1] https://github.com/wso2/identity-apps/blob/master/features/admin.organizations.v1/api/organization-role.ts#L201